src/neo4j/_async/work/session.py

```diff
@@ -353,6 +353,7 @@ async def last_bookmark(self) -> t.Optional[str]:
             return self._bookmarks[-1]
         return None
 
+    @AsyncNonConcurrentMethodChecker.non_concurrent_method
```

Comment:
What does this decorator means?

Reply:
It's coming from this PR https://github.com/neo4j/neo4j-python-driver/pull/989

The driver offers an optional debug mode where it tries to detect concurrent usage of objects that are not safe to use concurrently.

Reply:
thanks